index.qmd

```diff
@@ -0,0 +1,234 @@
+    
+    Alternatively, you can manually create the `environment.yml` file in the root of your project directory, where it would follow this structure: 
+
+    ```{.python}
+    name: my_env
+    dependencies:
+      - python=3.7
+      - pandas=0.24
+      - numpy=1.21
+    ```
+
+    If you installed packages using `pip`, the `environment.yml` file would follow this structure: 
+    
+    ```{.python}
+    name: my_env
+    dependencies:
+      - python=3.7
+      - pandas=0.24
+      - numpy=1.21
+      - pip=19.1
+      - pip:
+        - awscli==1.16
+        - kaggle==1.5
+    ```
+
+    Note that packages installed using `pip` use `==` to specify the version number, while packages installed from `conda` use `=`. 
+
+5. **Share** the snapshot by sending the `environment.yml` file to GitHub. 
+
+6. **Restore** and activate the snapshot of the environment on another computer: 
+
+    ```{.python}
+    conda env create      
+    conda activate my_env
+    ```
+
+    The `conda env create` syntax assumes you have a file called `environment.yml` in your working directory. While you could give this file a different name, we recommend sticking with the convention of naming the file `environment.yml` and placing it in the root of your project's directory. 
+
+7. **Repeat** the process. As you and your collaborators install, update, and remove packages, repeat steps 4-6 to save and load the state of your project to the `environment.yml` file across computers. 
+
+:::
+
+## Additional resources 
+
+::: {.panel-tabset}
+
+### R
+
+For more information, we recommend the official [Introduction to renv](https://rstudio.github.io/renv/articles/renv.html#package-sources) and [Collaborating with renv](https://rstudio.github.io/renv/articles/collaborating.html) vignettes. 
+
+### Python 
+
+We recommend keeping the official `conda` [cheat sheet](https://docs.conda.io/projects/conda/en/4.6.0/_downloads/52a95608c49671267e40c689e0bc00ca/conda-cheatsheet.pdf) handy.
+In addition to the commands included above, a few other commonly used `conda` commands include: 
+
+* `conda deactivate` deactivates the current environment  
+* `conda list` lists all packages in the current environment  
+* `conda env list` lists all environments (with the current active environment asterisked)  
+* `conda env remove -n my_env` deletes the `my_env` environment 
+
+The official [documentation](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html) and [this guide](https://carpentries-incubator.github.io/introduction-to-conda-for-data-scientists/04-sharing-environments/index.html) from the Carpentries are also helpful resources. 
+
+:::
+
+## Getting help 
+
+Adding a new tool to your workflow can be hard, but we're here to help! Feel free to email Erika Tyagi (etyagi@urban.org) or drop a message in the #reproducible-research Slack channel with any questions or if you run into issues. 
```

Comment:
I'd love to add at least one other person to this list – ideally someone with prior experience with both `conda` and `renv`?